.travis.yml

```diff
@@ -0,0 +1,13 @@
+android:
+  components:
+    # Update Android SDK Tools
+    - tools
+
+    - build-tools-27.1.1
+    - android-27
+
+    # Support library
+    - extra-android-support
+    - extra-android-m2repository
+script:
+  - chmod +x ./gradlew
```

Comment:
Please parse yml files through YAML Formatter, the merged .travis.yml had basic syntax issues. Please make sure it passes YAML Lint before committing further changes.

YML is a very unforgiving language, it looks deeply into indentation and stuff just like Python. Be sure, and careful :)